tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3784,20 +3815,52 @@ <h2>Show similarity to selected datapoint</h2>
           }
         },
 
+        isSameInferenceClass_: function(val1, val2) {
+          return this.isRegression_(this.modelType)
+            ? Math.abs(val1 - val2) < this.minCounterfactualValueDist
+            : val1 == val2;
+        },
+
+        adjustMaxCounterfactualValueDist_: function(selected, valueStr) {
+          this.maxCounterfactualValueDist = Math.max(
+            this.distanceStats_[valueStr].max -
+              this.visdata[selected][valueStr],
+            this.visdata[selected][valueStr] - this.distanceStats_[valueStr].min
+          );
+        },
+
+        adjustMinCounterfactualValueDist_: function() {
+          const valueStr = this.strWithModelName_(
+            inferenceValueStr,
```

Comment:
Where does this symbol `inferenceValueStr` come from? 

Ah, it is a global variable. Do action required but all module level constant should be ALL_CAPS :(

Reply:
Should I send a separate PR converting them to ALL_CAPS?

Reply:
we can do a separate PR changing all our model level constants to CAPS. no need in this review.